docs/src/pages/reference/configuration/hooks.md

```diff
@@ -20,3 +20,18 @@ module.exports = {
   },
 };
 ```
+
+If you don't want to inject the generated files into the command, you can use `afterAllFilesWrite` with an object:
+
+```js
+module.exports = {
+  petstore: {
+    hooks: {
+      afterAllFilesWrite: {
```

Comment:
Shouldn't it be `afterAllFilesWritten` ?

Reply:
I think too

Reply:
I took care of it

Reply:
@melloware @anymaniax 

Into code I saw `afterAllFilesWrite` and not `afterAllFilesWritten`

See :
- https://github.com/anymaniax/orval/blob/master/packages/orval/src/write-specs.ts#L150
- https://github.com/anymaniax/orval/blob/f529b9606165d9e5775ed9b65fc05b4187d6c327/packages/core/src/types.ts#L396

Reply:
Good catch looks like something got messed up with the PR.

Reply:
@Arkanii I just submitted a new PR if you want to review!